meson.build

```diff
@@ -1080,6 +1081,26 @@ if targetos == 'linux' and (have_system or have_tools)
                        required: get_option('libudev'))
 endif
 
+symcc_backend = get_option('symcc_backend')
+
+if not fs.is_file('subprojects/symcc-rt/RuntimeCommon.h')
+  error_msg='''The SymCC source path "@0@" is incorrect.
+Did you initialize symcc-rt subrepository? (git submodule update --init --recursive subprojects/symcc-rt)'''.format('subprojects/symcc')
```

Comment:
What's the reason for `.format(...)` if we're inserting a constant? And isn't the constant a lie? :grin: We're checking `subprojects/symcc-rt`, not `subprojects/symcc`.

Reply:
It was an old change I did and forgot to remove. Originally, it was not fixed like this and it was possible to set the path of the subproject.
It doesn't really make sense anymore indeed.